geomstats/geometry/special_orthogonal.py

```diff
@@ -43,64 +43,30 @@ def __init__(self, n):
         self.bi_invariant_metric = BiInvariantMetric(group=self)
 
     def belongs(self, point):
-        """Check whether point is an orthogonal matrix.
-
-        Parameters
-        ----------
-        point : array-like, shape=[..., n, n]
-            Point to check.
-
-        Returns
-        -------
-        belongs : array-like, shape=[...,]
-            Boolean evaluating if point belongs to SO(n).
-        """
+        """Check whether point is an orthogonal matrix."""
```

Comment:
Maybe a GitHub problem here: this docstring seems to have disappeared.

Reply:
It's come back...